packages/flutter_tools/lib/src/build_runner/build_script.dart

```diff
@@ -41,6 +41,19 @@ const String jsSourceMapExtension = '.ddc.js.map';
 const String kReleaseFlag = 'release';
 const String kProfileFlag = 'profile';
 
+// A minimum set of libraries to skip checks for to keep the examples compiling
+// until we make a decision on whether to support dart:io on the web.
```

Comment:
Is there a github issue for the dart:io decision?

Reply:
Updated with the issue tracking the primary API decision for Platform